msf_updates/src/position_pose_msf/position_pose_sensormanager.h

```diff
@@ -186,8 +186,15 @@ class PositionPoseSensorManager : public msf_core::MSF_SensorManagerROS<
     Eigen::Quaterniond yawq(cos(yawinit / 2), 0, 0, sin(yawinit / 2));
     yawq.normalize();
 
-    q = yawq;
-    q_wv = (q * q_ic * q_vc.conjugate()).conjugate();
+    q_wv = yawq.conjugate();
+
+    if (q_vc.w() == 1) {  // If there is no pose measurement, only apply q_wv.
+      q = q_wv;
+    } else {  // If there is a pose measurement, apply q_ic and q_wv to get initial attitud
+      q = (q_ic * q_vc.conjugate() * q_wv).conjugate();
```

Comment:
yes, that is indeed a conflict between notation and usage. There are several things that need to be fixed:
1) in the computation of the residual for orientation, the q_wv needs to be inverted.
2) in the computation of the residual for position the transpose on C_wv needs to be removed and then the conjugate removed in the computation of C_wv.
3) The use of C_q in ```C_q.transpose() * p_ic``` is also not consistent since there is another wrong ```conjugate``` in [L299](https://github.com/ethz-asl/ethzasl_msf/blob/master/msf_updates/include/msf_updates/pose_sensor_handler/pose_measurement.h#L299)

it seems I overlooked these when I did the big cleanup from the old filter version... :(